.travis.yml

```diff
@@ -0,0 +1,40 @@
+language: python
```

Comment:
Is this `.travis.yml` used?  I thought I converted all the scikit-tda repos to use github actions now.

Reply:
Artifact, I'll remove. This PR had updated .travis.yml before travis integration was removed. So I think the rebase re-added the file.